experimental/examples/attributable-map-demo/src/App.tsx

```diff
@@ -0,0 +1,88 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import React, { useEffect, useState } from "react";
+import { AttributableMap } from "@fluid-experimental/attributable-map";
+import { StaticCodeLoader, TinyliciousModelLoader } from "@fluid-example/example-utils";
+import { AttributionKey } from "@fluidframework/runtime-definitions";
+import {
+	AttributableMapPageContainerRuntimeFactory,
+	IAttributableMapPageAppModel,
+} from "./container";
+
+const timeKey = "time-key";
+
+const getMyMap = async () => {
+	const tinyliciousModelLoader = new TinyliciousModelLoader<IAttributableMapPageAppModel>(
+		new StaticCodeLoader(new AttributableMapPageContainerRuntimeFactory()),
+	);
+
+	let id: string;
+	let model: IAttributableMapPageAppModel;
+
+	if (location.hash.length === 0) {
+		const createResponse = await tinyliciousModelLoader.createDetached("1.0");
+		model = createResponse.model;
+		id = await createResponse.attach();
+	} else {
+		id = location.hash.substring(1);
+		model = await tinyliciousModelLoader.loadExisting(id);
+	}
+
+	// update the browser URL and the window title with the actual container ID
+	location.hash = id;
+	document.title = id;
+
+	return model.attributableMapPage.map;
+};
+
+const App: React.FC = () => {
+	const [fluidMap, setFluidMap] = useState<AttributableMap>();
+	useEffect(() => {
+		void getMyMap().then((myMap) => setFluidMap(myMap));
```

Comment:
Take a look at a few of the other examples (e.g. external-views) for a flow that loads the container/DataObject outside of the view.  This works a lot nicer to avoid blocking data load on view render, and makes the view simpler since it doesn't have to perform load or support render before load.